src/include/uapi/ipc.h

```diff
@@ -500,7 +500,8 @@ struct sof_ipc_stream_params {
 	enum sof_ipc_stream_direction direction;
 	enum sof_ipc_frame frame_fmt;
 	enum sof_ipc_buffer_format buffer_fmt;
-	uint32_t stream_tag;
+	uint32_t host_dma_ch;
+	uint32_t be_dma_ch;
```

Comment:
same comment as for the driver, I don't see why the link DMA info be_dma_ch is passed in the ipc structure used for the host.

Reply:
thanks, please see my comment in driver side